### PR TITLE
#1 Set up the Laravel API project

### LINKED ISSUE #1: [BE] Setup the Laravel API

### PR DESCRIPTION
**Pull request by davorminchorov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,66 @@
-# api
-The API project for LearnHub.mk
+<p align="center"><a href="https://laravel.com" target="_blank"><img src="https://raw.githubusercontent.com/laravel/art/master/logo-lockup/5%20SVG/2%20CMYK/1%20Full%20Color/laravel-logolockup-cmyk-red.svg" width="400" alt="Laravel Logo"></a></p>
```
Comment (RevanchistX) [inline README.md:1]: LGTM

**Review by RevanchistX — APPROVED** [review]
Davor setup this project magically. Well done, Davor!

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (davorminchorov) [inline .editorconfig:12]: Test

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+APP_ENV=local
+APP_KEY=
+APP_DEBUG=true
+APP_URL=http://localhost
+
+LOG_CHANNEL=stack
+LOG_DEPRECATIONS_CHANNEL=null
+LOG_LEVEL=debug
+
+DB_CONNECTION=mysql
```
Comment (davorminchorov) [inline .env.example:11]: Add another comment

**Pull request merged**